Tests/TransferMethod/ListTransferMethodPresenterTests.swift

```diff
@@ -178,7 +196,13 @@ class ListTransferMethodPresenterTests: XCTestCase {
                                                    transferMethodProfileType: "INDIVIDUAL")
             .build()
         bankCard.setField(key: "token", value: "trm-123456789")
-        let transferMethods = [bankAccount, bankCard]
+
+        let payPalAccount = HyperwalletPayPalAccount.Builder(transferMethodCountry: "US",
```

Comment:
We should use ``` HyperwalletPayPalAccount.Builder(token: "trm-123456789").build()```

Reply:
We can't. We need country and currency as well